source/hic_hal/stm32/stm32h743ii/extended_features/include/udb_extended_features_task.h

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Description:
+ *    Create an auxiliary task so that we don't need to put UDB's logic in the main task.
+ *
+ */
+#ifndef _UDB_EXTENDED_FEATURES_TASK_INCLUDED_H_
+#define _UDB_EXTENDED_FEATURES_TASK_INCLUDED_H_
+
+#include "cmsis_os2.h"
+
+#define UDB_EXTENDED_FEATURES_TASK_STACK          (800)
```

Comment:
I think these defines can all be moved into udb_extended_features_task.c where they are used. No need to cause other files to include more definitions than necessary.